packages/block-library/src/column/edit.js

```diff
@@ -63,7 +64,7 @@ function ColumnEdit( {
 				</PanelBody>
 			</InspectorControls>
 			<InnerBlocks
-				templateLock={ false }
+				templateLock={ parentLock }
```

Comment:
Although this locking is inherited for the innerBlocks mechanism it looks like this locking was explicitly set. There are some differences between explicitly set lockings and inherited lockings namely how the template sync is applied. Given that the column does not set a template it is safe to do this, but people in the community may watch this as an example.
Maybe we can include a comment.
```
// It is safe to set an "inherited" locking explicitly because no template is set on the column.
// If a template was set and parent lock was equal to all the local template set would be forced and other templates passed e.g: via CPT would be ignored.
```